server/database/models/AssignedChore.js

```diff
@@ -1,5 +1,5 @@
-const Sequelize = require('./node_modules/sequelize');
```

Comment:
whats going on here?

Reply:
looks like this was in a previous commit, not sure why its showing up.  Will double chekc the push this time to make sure it goes through